LICENSE

```diff
@@ -1,7 +1,7 @@
 Node.js is licensed for use as follows:
 
 """
-Copyright Node.js contributors. All rights reserved.
+Copyright Joyent, Inc. and other Node.js contributors. All rights reserved.
```

Comment:
Fair enough. My interpretation would be that the first block applies to files that don't originate from `https://github.com/joyent/node` (as that is what the second block is for). But IANAL.